packages/admob/android/src/main/java/io/invertase/firebase/admob/ReactNativeFirebaseAdMobCommon.java

```diff
@@ -39,6 +44,34 @@
 
 public class ReactNativeFirebaseAdMobCommon {
 
+  static AdSize getAdSizeForAdaptiveBanner(ReactViewGroup reactViewGroup) {
+
+    try {
+      Display display = Objects.requireNonNull(((ReactContext) reactViewGroup.getContext()).getCurrentActivity()).getWindowManager().getDefaultDisplay();
+
+      DisplayMetrics outMetrics = new DisplayMetrics();
+      display.getMetrics(outMetrics);
+
+      float widthPixels = outMetrics.widthPixels;
+      float density = outMetrics.density;
+
+      int adWidth = (int) (widthPixels / density);
+
+      return AdSize.getCurrentOrientationAnchoredAdaptiveBannerAdSize(reactViewGroup.getContext(), adWidth);
+    } catch (Exception e) {
+      return AdSize.BANNER;
+    }
+
+  }
+
+  static AdSize getAdSize(String preDefinedAdSize, ReactViewGroup reactViewGroup) {
+    if (preDefinedAdSize.equals("ADAPTIVE_BANNER")) {
+      return getAdSizeForAdaptiveBanner(reactViewGroup);
```

Comment:
```suggestion
      return ReactNativeFirebaseAdMobCommon.getAdSizeForAdaptiveBanner(reactViewGroup);
```

consistently use static references to the methods if they are static